doc/docs/getting_started.md

```diff
@@ -13,161 +11,559 @@ dnf copr enable mperina/hirte-snapshot centos-stream-9
 
 This should no longer be necessary once BlueChi is made available in EPEL.
 
-On the laptop install both bluechi and the bluechi-agent via this command:
+All components of BlueChi can be installed via:
```

Comment:
We don't need to mention snapshot repo here, bluechi is already available in EPEL:

```shell
$ dnf info bluechi
Available Packages
Name         : bluechi
Version      : 0.5.0
Release      : 1.el9
Architecture : x86_64
Size         : 71 k
Source       : bluechi-0.5.0-1.el9.src.rpm
Repository   : epel
Summary      : A systemd service controller for multi-nodes environments
URL          : https://github.com/containers/bluechi
License      : LGPL-2.1-or-later AND CC0-1.0
Description  : BlueChi is a systemd service controller for multi-nodes environements with a
             : predefined number of nodes and with a focus on highly regulated environment
             : such as those requiring functional safety (for example in cars).
             : 
             : This package contains the controller and command line tool.

```

Reply:
Good point! I changed it accordingly. Could you have another look? @mwperina